lib/DBIx/Class/Helper/ResultSet/Shortcut/ResultsExist.pm

```diff
@@ -14,7 +14,7 @@ sub results_exist_as_query {
    } )->as_query;
 
 
-   $$reified->[0] = "( SELECT EXISTS $$reified->[0] )";
+   $$reified->[0] = "( SELECT EXISTS $$reified->[0] ) as forty_two";
```

Comment:
@rabbiveesh doing this here will in fact break composability in unforseseen ways. Instead you need to do something like: https://github.com/frioux/DBIx-Class-Helpers/issues/54#issuecomment-563376551

Aliasing a subselect at the point if generation is problematic, as you preclude using multiple such selects in the same "context".

Reply:
kkk, i changed it